libs/langchain/langchain/chat_models/openai.py

```diff
@@ -374,7 +374,13 @@ def _stream(
             default_chunk_class = chunk.__class__
             yield ChatGenerationChunk(message=chunk)
             if run_manager:
-                run_manager.on_llm_new_token(chunk.content)
+                chunk_content = chunk.additional_kwargs.get("function_call", None)
+                token = (
+                    chunk_content["arguments"]
+                    if chunk_content is not None
+                    else chunk.content
```

Comment:
What if the chunk has both content and arguments?

Reply:
From what I've seen in testing, this does not happen. When the chain is called as part of a function/tool call, the output is in the form of a function, and the "standard" chunk content is empty.

Reply:
@AI-Chef To be fair, that is probably a safer way of handling this for the base class as well! I'll push a commit to fix that in a little bit 👌

Reply:
The funciton calling endpoint does sometimes return both, and in any case it wouldn't be a behavior we could bank on long-term, so we added the whole chunk to the callback